config/settings.yml

```diff
@@ -62,3 +62,6 @@ teacher_training_api:
   
 environment:
   name: qa
+
+slack: 
+  default_channel: "#twd_publish_register_tech"
```

Comment:
Needs a newline. I wonder if `default_channel` is the right thing to call it too. `developer_alerts_channel` or something maybe? I'm thinking if we add other alerts that still makes sense and then if we add other types of messages it's obvious that we might need to think of another channel. e.g. if we added stats info we might have `product_stats_channel` and it might go to twd_publish_register or something like that.